15.0.x server tests failing with missing file infinispan-custom.xml

### Description

```java
2025-05-20T12:34:59.3530918Z [ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:3.1.0:run (generate-distros-and-configs) on project infinispan-server-integration: An Ant BuildException has occured: The following error occurred while executing this line:
2025-05-20T12:34:59.3534619Z [ERROR] /home/runner/work/infinispan/infinispan/test_dir/infinispan/server/runtime/src/main/ant/infinispan-server.xml:19: Warning: Could not find file /home/runner/work/infinispan/infinispan/test_dir/infinispan/integrationtests/server-integration/server-integration-commons/target/test-classes/infinispan-custom.xml to copy.
2025-05-20T12:34:59.3539233Z [ERROR] around Ant part ...<ant antfile="/home/runner/work/infinispan/infinispan/test_dir/infinispan/integrationtests/server-integration/../../server/runtime/src/main/ant/infinispan-server.xml" target="create-distro">... @ 4:195 in /home/runner/work/infinispan/infinispan/test_dir/infinispan/integrationtests/server-integration/target/antrun/build-main.xml
2025-05-20T12:34:59.3541881Z [ERROR] -> [Help 1]
2025-05-20T12:34:59.3544117Z org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:3.1.0:run (generate-distros-and-configs) on project infinispan-server-integration: An Ant BuildException has occured: The following error occurred while executing this line:
2025-05-20T12:34:59.3548422Z /home/runner/work/infinispan/infinispan/test_dir/infinispan/server/runtime/src/main/ant/infinispan-server.xml:19: Warning: Could not find file /home/runner/work/infinispan/infinispan/test_dir/infinispan/integrationtests/server-integration/server-integration-commons/target/test-classes/infinispan-custom.xml to copy.
2025-05-20T12:34:59.3552882Z around Ant part ...<ant antfile="/home/runner/work/infinispan/infinispan/test_dir/infinispan/integrationtests/server-integration/../../server/runtime/src/main/ant/infinispan-server.xml" target="create-distro">... @ 4:195 in /home/runner/work/infinispan/infinispan/test_dir/infinispan/integrationtests/server-integration/target/antrun/build-main.xml
2025-05-20T12:34:59.3556182Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:347)
2025-05-20T12:34:59.3557480Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:330)
2025-05-20T12:34:59.3558721Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:213)
2025-05-20T12:34:59.3559943Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:175)
2025-05-20T12:34:59.3561179Z     at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:76)
2025-05-20T12:34:59.3562404Z     at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:163)
2025-05-20T12:34:59.3563801Z     at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
2025-05-20T12:34:59.3565219Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:160)
2025-05-20T12:34:59.3566951Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
2025-05-20T12:34:59.3568607Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
2025-05-20T12:34:59.3570393Z     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
2025-05-20T12:34:59.3572044Z     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
2025-05-20T12:34:59.3573379Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
2025-05-20T12:34:59.3574360Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
2025-05-20T12:34:59.3575589Z     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
2025-05-20T12:34:59.3576526Z     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:827)
2025-05-20T12:34:59.3577420Z     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:272)
2025-05-20T12:34:59.3578256Z     at org.apache.maven.cli.MavenCli.main (MavenCli.java:195)
2025-05-20T12:34:59.3579307Z     at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
2025-05-20T12:34:59.3580421Z     at java.lang.reflect.Method.invoke (Method.java:580)
2025-05-20T12:34:59.3581492Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
2025-05-20T12:34:59.3582738Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
2025-05-20T12:34:59.3584011Z     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
2025-05-20T12:34:59.3585267Z     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
2025-05-20T12:34:59.3586714Z     at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
2025-05-20T12:34:59.3587803Z     at java.lang.reflect.Method.invoke (Method.java:580)
2025-05-20T12:34:59.3588814Z     at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
2025-05-20T12:34:59.3590177Z     at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
2025-05-20T12:34:59.3591380Z     at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
2025-05-20T12:34:59.3592996Z Caused by: org.apache.maven.plugin.MojoExecutionException: An Ant BuildException has occured: The following error occurred while executing this line:
2025-05-20T12:34:59.3596551Z /home/runner/work/infinispan/infinispan/test_dir/infinispan/server/runtime/src/main/ant/infinispan-server.xml:19: Warning: Could not find file /home/runner/work/infinispan/infinispan/test_dir/infinispan/integrationtests/server-integration/server-integration-commons/target/test-classes/infinispan-custom.xml to copy.
2025-05-20T12:34:59.3600923Z around Ant part ...<ant antfile="/home/runner/work/infinispan/infinispan/test_dir/infinispan/integrationtests/server-integration/../../server/runtime/src/main/ant/infinispan-server.xml" target="create-distro">... @ 4:195 in /home/runner/work/infinispan/infinispan/test_dir/infinispan/integrationtests/server-integration/target/antrun/build-main.xml
2025-05-20T12:34:59.3603791Z     at org.apache.maven.plugins.antrun.AntRunMojo.execute (AntRunMojo.java:318)
2025-05-20T12:34:59.3605293Z     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:126)
2025-05-20T12:34:59.3607091Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:342)
2025-05-20T12:34:59.3608556Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:330)
2025-05-20T12:34:59.3609969Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:213)
2025-05-20T12:34:59.3611388Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:175)
2025-05-20T12:34:59.3612789Z     at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:76)
2025-05-20T12:34:59.3614044Z     at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:163)
2025-05-20T12:34:59.3615625Z     at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
2025-05-20T12:34:59.3617075Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:160)
2025-05-20T12:34:59.3618551Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
2025-05-20T12:34:59.3620507Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
2025-05-20T12:34:59.3622471Z     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
2025-05-20T12:34:59.3624323Z     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
2025-05-20T12:34:59.3625730Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
2025-05-20T12:34:59.3626700Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
2025-05-20T12:34:59.3627646Z     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
2025-05-20T12:34:59.3628554Z     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:827)
2025-05-20T12:34:59.3629409Z     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:272)
2025-05-20T12:34:59.3630247Z     at org.apache.maven.cli.MavenCli.main (MavenCli.java:195)
2025-05-20T12:34:59.3631366Z     at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
2025-05-20T12:34:59.3632451Z     at java.lang.reflect.Method.invoke (Method.java:580)
2025-05-20T12:34:59.3633490Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
2025-05-20T12:34:59.3634717Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
2025-05-20T12:34:59.3636123Z     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
2025-05-20T12:34:59.3637453Z     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
2025-05-20T12:34:59.3638747Z     at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
2025-05-20T12:34:59.3639830Z     at java.lang.reflect.Method.invoke (Method.java:580)
2025-05-20T12:34:59.3640812Z     at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
2025-05-20T12:34:59.3641969Z     at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
2025-05-20T12:34:59.3643103Z     at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
2025-05-20T12:34:59.3644294Z Caused by: org.apache.tools.ant.BuildException: The following error occurred while executing this line:
2025-05-20T12:34:59.3647274Z /home/runner/work/infinispan/infinispan/test_dir/infinispan/server/runtime/src/main/ant/infinispan-server.xml:19: Warning: Could not find file /home/runner/work/infinispan/infinispan/test_dir/infinispan/integrationtests/server-integration/server-integration-commons/target/test-classes/infinispan-custom.xml to copy.
2025-05-20T12:34:59.3650025Z     at org.apache.tools.ant.ProjectHelper.addLocationToBuildException (ProjectHelper.java:582)
2025-05-20T12:34:59.3651197Z     at org.apache.tools.ant.taskdefs.Ant.execute (Ant.java:440)
2025-05-20T12:34:59.3652173Z     at org.apache.tools.ant.UnknownElement.execute (UnknownElement.java:299)
2025-05-20T12:34:59.3653426Z     at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
2025-05-20T12:34:59.3654551Z     at java.lang.reflect.Method.invoke (Method.java:580)
2025-05-20T12:34:59.3655662Z     at org.apache.tools.ant.dispatch.DispatchUtils.execute (DispatchUtils.java:99)
2025-05-20T12:34:59.3656643Z     at org.apache.tools.ant.Task.perform (Task.java:350)
2025-05-20T12:34:59.3657455Z     at org.apache.tools.ant.Target.execute (Target.java:449)
2025-05-20T12:34:59.3658334Z     at org.apache.tools.ant.Target.performTasks (Target.java:470)
2025-05-20T12:34:59.3659321Z     at org.apache.tools.ant.Project.executeSortedTargets (Project.java:1401)
2025-05-20T12:34:59.3660336Z     at org.apache.tools.ant.Project.executeTarget (Project.java:1374)
2025-05-20T12:34:59.3661357Z     at org.apache.maven.plugins.antrun.AntRunMojo.execute (AntRunMojo.java:297)
2025-05-20T12:34:59.3662667Z     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:126)
2025-05-20T12:34:59.3664075Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:342)
2025-05-20T12:34:59.3665809Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:330)
2025-05-20T12:34:59.3667171Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:213)
2025-05-20T12:34:59.3668403Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:175)
2025-05-20T12:34:59.3669622Z     at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:76)
2025-05-20T12:34:59.3670831Z     at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:163)
2025-05-20T12:34:59.3672213Z     at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
2025-05-20T12:34:59.3673610Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:160)
2025-05-20T12:34:59.3675048Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
2025-05-20T12:34:59.3676872Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
2025-05-20T12:34:59.3678659Z     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
2025-05-20T12:34:59.3680281Z     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
2025-05-20T12:34:59.3681433Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
2025-05-20T12:34:59.3682487Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
2025-05-20T12:34:59.3683453Z     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
2025-05-20T12:34:59.3684352Z     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:827)
2025-05-20T12:34:59.3685214Z     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:272)
2025-05-20T12:34:59.3686306Z     at org.apache.maven.cli.MavenCli.main (MavenCli.java:195)
2025-05-20T12:34:59.3687488Z     at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
2025-05-20T12:34:59.3688638Z     at java.lang.reflect.Method.invoke (Method.java:580)
2025-05-20T12:34:59.3689794Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
2025-05-20T12:34:59.3691080Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
2025-05-20T12:34:59.3692496Z     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
2025-05-20T12:34:59.3693774Z     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
2025-05-20T12:34:59.3695092Z     at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
2025-05-20T12:34:59.3696371Z     at java.lang.reflect.Method.invoke (Method.java:580)
2025-05-20T12:34:59.3697407Z     at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
2025-05-20T12:34:59.3698618Z     at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
2025-05-20T12:34:59.3699749Z     at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
2025-05-20T12:34:59.3702070Z Caused by: org.apache.tools.ant.BuildException: Warning: Could not find file /home/runner/work/infinispan/infinispan/test_dir/infinispan/integrationtests/server-integration/server-integration-commons/target/test-classes/infinispan-custom.xml to copy.
2025-05-20T12:34:59.3704277Z     at org.apache.tools.ant.taskdefs.Copy.copySingleFile (Copy.java:643)
2025-05-20T12:34:59.3705249Z     at org.apache.tools.ant.taskdefs.Copy.execute (Copy.java:463)
2025-05-20T12:34:59.3706375Z     at org.apache.tools.ant.UnknownElement.execute (UnknownElement.java:299)
2025-05-20T12:34:59.3707607Z     at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
2025-05-20T12:34:59.3708700Z     at java.lang.reflect.Method.invoke (Method.java:580)
2025-05-20T12:34:59.3709761Z     at org.apache.tools.ant.dispatch.DispatchUtils.execute (DispatchUtils.java:99)
2025-05-20T12:34:59.3710717Z     at org.apache.tools.ant.Task.perform (Task.java:350)
2025-05-20T12:34:59.3711614Z     at org.apache.tools.ant.Target.execute (Target.java:449)
2025-05-20T12:34:59.3712475Z     at org.apache.tools.ant.Target.performTasks (Target.java:470)
2025-05-20T12:34:59.3713415Z     at org.apache.tools.ant.Project.executeSortedTargets (Project.java:1401)
2025-05-20T12:34:59.3714624Z     at org.apache.tools.ant.helper.SingleCheckExecutor.executeTargets (SingleCheckExecutor.java:36)
2025-05-20T12:34:59.3745654Z     at org.apache.tools.ant.Project.executeTargets (Project.java:1264)
2025-05-20T12:34:59.3746695Z     at org.apache.tools.ant.taskdefs.Ant.execute (Ant.java:437)
2025-05-20T12:34:59.3747630Z     at org.apache.tools.ant.UnknownElement.execute (UnknownElement.java:299)
2025-05-20T12:34:59.3748840Z     at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
2025-05-20T12:34:59.3749948Z     at java.lang.reflect.Method.invoke (Method.java:580)
2025-05-20T12:34:59.3750832Z     at org.apache.tools.ant.dispatch.DispatchUtils.execute (DispatchUtils.java:99)
2025-05-20T12:34:59.3751720Z     at org.apache.tools.ant.Task.perform (Task.java:350)
2025-05-20T12:34:59.3752530Z     at org.apache.tools.ant.Target.execute (Target.java:449)
2025-05-20T12:34:59.3753422Z     at org.apache.tools.ant.Target.performTasks (Target.java:470)
2025-05-20T12:34:59.3754411Z     at org.apache.tools.ant.Project.executeSortedTargets (Project.java:1401)
2025-05-20T12:34:59.3761340Z     at org.apache.tools.ant.Project.executeTarget (Project.java:1374)
2025-05-20T12:34:59.3762511Z     at org.apache.maven.plugins.antrun.AntRunMojo.execute (AntRunMojo.java:297)
2025-05-20T12:34:59.3763849Z     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:126)
2025-05-20T12:34:59.3765272Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:342)
2025-05-20T12:34:59.3787802Z     at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:330)
2025-05-20T12:34:59.3789096Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:213)
2025-05-20T12:34:59.3790347Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:175)
2025-05-20T12:34:59.3791589Z     at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:76)
2025-05-20T12:34:59.3792960Z     at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:163)
2025-05-20T12:34:59.3794336Z     at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
2025-05-20T12:34:59.3804809Z     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:160)
2025-05-20T12:34:59.3806489Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
2025-05-20T12:34:59.3808139Z     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
2025-05-20T12:34:59.3809897Z     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
2025-05-20T12:34:59.3811541Z     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
2025-05-20T12:34:59.3812719Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
2025-05-20T12:34:59.3813722Z     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
2025-05-20T12:34:59.3814700Z     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
2025-05-20T12:34:59.3860544Z     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:827)
2025-05-20T12:34:59.3861522Z     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:272)
2025-05-20T12:34:59.3862476Z     at org.apache.maven.cli.MavenCli.main (MavenCli.java:195)
2025-05-20T12:34:59.3863833Z     at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
2025-05-20T12:34:59.3864972Z     at java.lang.reflect.Method.invoke (Method.java:580)
2025-05-20T12:34:59.3867114Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
2025-05-20T12:34:59.3868528Z     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
2025-05-20T12:34:59.3869816Z     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
2025-05-20T12:34:59.3871085Z     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
2025-05-20T12:34:59.3872375Z     at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
2025-05-20T12:34:59.3873474Z     at java.lang.reflect.Method.invoke (Method.java:580)
2025-05-20T12:34:59.3874491Z     at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
2025-05-20T12:34:59.3875909Z     at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
2025-05-20T12:34:59.3877090Z     at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)

```


Backport of https://issues.redhat.com/browse/ISPN-16712 required

### Implementation ideas

2a38928b196dd9cbace8245d4d86ddd4c15d3551

--- a/integrationtests/server-integration/pom.xml
+++ b/integrationtests/server-integration/pom.xml
@@ -74,65 +74,6 @@
                     </dependency>
                 </dependencies>
             </plugin>
-            <plugin>
-                <artifactId>maven-antrun-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>generate-distros-and-configs</id>
-                        <goals>
-                            <goal>run</goal>
-                        </goals>
-                        <phase>package</phase>
-                        <configuration>
-                            <skip>${skipTests}</skip>
-                            <target>
-                                <ant antfile="${ispnserver.project.dir}/src/main/ant/infinispan-server.xml" target="create-distro">
-                                    <property name="server.build.dist" value="${server.build.dist}" />
-                                    <property name="server.dist" value="${ispnserver.dist}" />
-                                    <property name="ispn.config.file" value="${ispn.config.file}" />
-                                </ant>
-                            </target>
-                        </configuration>
-                    </execution>
-                    <execution>
-                        <id>infinispan-server-startup</id>
-                        <phase>pre-integration-test</phase>
-                        <goals>
-                            <goal>run</goal>
-                        </goals>
-                        <configuration>
-                            <skip>${skipTests}</skip>
-                            <target>
-                                <ant antfile="${ispnserver.project.dir}/src/main/ant/infinispan-server.xml" target="kill-server">
-                                    <property name="hotrod.port" value="11222" />
-                                </ant>
-                                <ant antfile="${ispnserver.project.dir}/src/main/ant/infinispan-server.xml" target="start-server">
-                                    <property name="server.dist" value="${ispnserver.dist}" />
-                                    <property name="port.offset" value="0" />
-                                    <property name="hotrod.port" value="11222" />
-                                    <property name="jboss.node.name" value="ispn-server" />
-                                    <property name="jboss.config.file" value="infinispan-custom.xml" />
-                                </ant>
-                            </target>
-                        </configuration>
-                    </execution>
-                    <execution>
-                        <id>infinispan-server-shutdown</id>
-                        <phase>post-integration-test</phase>
-                        <goals>
-                            <goal>run</goal>
-                        </goals>
-                        <configuration>
-                            <skip>${skipTests}</skip>
-                            <target>
-                                <ant antfile="${ispnserver.project.dir}/src/main/ant/infinispan-server.xml" target="kill-server">
-                                    <property name="hotrod.port" value="11222" />
-                                </ant>
-                            </target>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
         </plugins>
     </build>
 </project>

--- a/integrationtests/server-integration/server-integration-commons/pom.xml
+++ b/integrationtests/server-integration/server-integration-commons/pom.xml
@@ -13,11 +13,6 @@
     <name>Infinispan Integration Tests Server Commons</name>
     <description>Infinispan Integration Tests Server Commons</description>
 
-    <properties>
-        <ispnserver.project.dir>${basedir}/../../../server/runtime</ispnserver.project.dir>
-        <ispn.config.file>${basedir}/../server-integration-commons/target/test-classes/infinispan-custom.xml</ispn.config.file>
-    </properties>
-
     <dependencies>
         <!-- the following dependency must be provided because each server ( WF, WF with modules or Tomcat )
         knows ( use as module or add a jar into war ) what should be done.
@@ -141,5 +136,4 @@
             </plugin>
         </plugins>
     </build>
-
 </project>

--- a/integrationtests/server-integration/third-party-server/pom.xml
+++ b/integrationtests/server-integration/third-party-server/pom.xml
@@ -154,6 +154,66 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>generate-distros-and-configs</id>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <phase>package</phase>
+                        <configuration>
+                            <skip>${skipTests}</skip>
+                            <target>
+                                <ant antfile="${ispnserver.project.dir}/src/main/ant/infinispan-server.xml" target="create-distro">
+                                    <property name="server.build.dist" value="${server.build.dist}" />
+                                    <property name="server.dist" value="${ispnserver.dist}" />
+                                    <property name="ispn.config.file" value="${ispn.config.file}" />
+                                </ant>
+                            </target>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>infinispan-server-startup</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <skip>${skipTests}</skip>
+                            <target>
+                                <ant antfile="${ispnserver.project.dir}/src/main/ant/infinispan-server.xml" target="kill-server">
+                                    <property name="hotrod.port" value="11222" />
+                                </ant>
+                                <ant antfile="${ispnserver.project.dir}/src/main/ant/infinispan-server.xml" target="start-server">
+                                    <property name="server.dist" value="${ispnserver.dist}" />
+                                    <property name="port.offset" value="0" />
+                                    <property name="hotrod.port" value="11222" />
+                                    <property name="jboss.node.name" value="ispn-server" />
+                                    <property name="jboss.config.file" value="infinispan-custom.xml" />
+                                </ant>
+                            </target>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>infinispan-server-shutdown</id>
+                        <phase>post-integration-test</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <skip>${skipTests}</skip>
+                            <target>
+                                <ant antfile="${ispnserver.project.dir}/src/main/ant/infinispan-server.xml" target="kill-server">
+                                    <property name="hotrod.port" value="11222" />
+                                </ant>
+                            </target>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
         </plugins>
     </build>
 

